bee-node/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[target.'cfg(feature = "trace")']
+rustflags = [ "--cfg", "tokio_unstable" ]
```

Comment:
Any reason this file is specifically in this folder ?

Reply:
Yep, I forgot to move it along with the refactor :trollface: